### PR TITLE
Support saddle finance airdrop

### LINKED ISSUE #3725: Support saddle finance airdrop
## Abstract

Seems saddle.finance is doing an airdrop. @banteg sent us this link: https://github.com/saddle-finance/saddle-retroactive-merkle-tree/blob/master/output/results.json

## Specification

Add it to the supported airdrops along with all the other ones.

### PR DESCRIPTION
**Pull request by Aragar199**
Closes #3725 

## Checklist

- [X] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by yabirgb** [comment]
Hello @Aragar199 seems that your PR got commits from develop combined with your commit.

File: rotkehlchen/constants/assets.py.template [hunk 1]
```diff
@@ -214,4 +214,5 @@ A_SX = EthereumToken('0x99fE3B1391503A1bC1788051347A1324bff41452')
 A_RSPT = EthereumToken('0x016bf078ABcaCB987f0589a6d3BEAdD4316922B0')
 A_FOX = EthereumToken('0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d')
 A_ENS = EthereumToken('0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72')
-A_PSP = EthereumToken('0xcAfE001067cDEF266AfB7Eb5A286dCFD277f3dE5')
\ No newline at end of file
+A_PSP = EthereumToken('0xcAfE001067cDEF266AfB7Eb5A286dCFD277f3dE5')
+A_SDL = EthereumToken('0xf1dc500fde233a4055e25e5bbf516372bc4f6871')
```
Comment (yabirgb) [inline rotkehlchen/constants/assets.py.template:218]: The reason the asset is not properly displayed is this initialization. The address is not checksummed and the asset is not correctly resolved. You need to change it to `0xf1Dc500FdE233A4055e25e5BbF516372BC4F6871` and then recreate the `assets.py` from this template

File: rotkehlchen/constants/assets.py [hunk 2]
```diff
@@ -146,8 +146,8 @@
 A_BSQ = Asset.initialize(
     identifier='BSQ',
     asset_type=AssetType.OTHER,
-    name="Bisq DAO Token",
-    symbol='BSQ',
+    name="BSQ",
+    symbol='Bisq DAO Token',
```
Comment (yabirgb) [inline rotkehlchen/constants/assets.py:150]: Hello @Aragar199 ! Thank you so much for fixing the issues but there is something else. This is some mistake I made in the SQL files but is already fixed. Will tell you how to fix for your database. Copy the `rotkehlchen/data/global.db` of the repo to the `global_data` folder for rotki in your system https://rotki.readthedocs.io/en/latest/usage_guide.html#rotki-data-directory (make a backup of the old one) This db contains the fix for the BSQ asset. Then recreate this file as you did before.

You can check the diff before commiting. The only things changing should be the header of this file with the new date and at the end the A_SDL asset.

Sorry for the inconvenience

Comment (Aragar199) [reply rotkehlchen/constants/assets.py]: Ah okay, no problem. It was an easy fix. Let me know if there is anything else!

**Review by yabirgb — APPROVED** [review]
Thank you @Aragar199 ! Looks good.

@LefterisJP When you can please merge the PR. Failing tests are:
- Two timeouts calling contracts
- NFT known issue with renamed collection

**Review by LefterisJP — APPROVED** [review]
LGTM, especially since @yabirgb approves

**Pull request merged**